molecule/telemetry/telemetry-ps/tests/test_telemetry_pillar.py

```diff
@@ -254,17 +254,18 @@ def test_ta_no_restart(host):
             ta_terminated_num = host.run(f'grep -c "Received signal: terminated, shutdow" {ta_log_file}')
             assert int(ta_terminated_num.stdout) == 0, (ta_terminated_num.stdout, ta_terminated_num.stderr)
 
+# Uncomment after https://perconadev.atlassian.net/browse/PKG-168 is fixed.
 # check that the old log file is not present after update and that its content is copied to the new log.  To be removed after 1.0.1-1 to 1.0.1-2 update
-def test_ta_update(host):
-    with host.sudo("root"):
-        if update == 'yes':
-            ta_started_num = host.run(f'grep -c "values from config:" {ta_log_file}')
-            assert int(ta_started_num.stdout) == 2, (ta_started_num.stdout, ta_started_num.stderr)
-            ta_terminated_num = host.run(f'grep -c "Received signal: terminated, shutdow" {ta_log_file}')
-            assert int(ta_terminated_num.stdout) == 1, (ta_terminated_num.stdout, ta_terminated_num.stderr)
-            assert not host.file('/var/log/percona/telemetry-agent.log').is_file
-        else:
-            pytest.skip("This check only for TA update")
+# def test_ta_update(host):
+#     with host.sudo("root"):
+#         if update == 'yes':
+#             ta_started_num = host.run(f'grep -c "values from config:" {ta_log_file}')
+#            assert int(ta_started_num.stdout) == 2, (ta_started_num.stdout, ta_started_num.stderr)
```

Comment:
Maybe instead of commenting out it would be better to use `assert int(ta_started_num.stdout) in [ 2, 3], (ta_started_num.stdout, ta_started_num.stderr)`